tests/test_image.py

```diff
@@ -122,6 +122,18 @@ def test_imnormalize(self):
         img = mmcv.imnormalize(rgbimg, self.mean, self.std, to_rgb=False)
         assert np.allclose(img, baseline)
 
+    def test_imnormalize_(self):
+        img_for_normalize = np.float32(self.img.copy())
+        rgbimg_for_normalize = np.float32(self.img[:, :, ::-1].copy())
+        baseline = (rgbimg_for_normalize - self.mean) / self.std
+        img = mmcv.imnormalize_(img_for_normalize, self.mean, self.std)
+        assert np.allclose(img_for_normalize, baseline)
+        assert id(img) == id(img_for_normalize)
```

Comment:
You may also assert assert id(img) != id(img_for_normalize) in `test_imnormalize`.